tests/test_project.py

```diff
@@ -288,31 +286,6 @@ def test_workspace_read_only_path(self):
         assert not os.path.isdir(self._tmp_wd)
         assert not os.path.isdir(self.project.workspace())
 
-    def test_find_job_ids(self):
-        statepoints = [{"a": i} for i in range(5)]
-        for sp in statepoints:
-            self.project.open_job(sp).document["b"] = sp["a"]
-        with pytest.deprecated_call():
-            assert len(statepoints) == len(list(self.project.find_job_ids()))
-            assert 1 == len(list(self.project.find_job_ids({"a": 0})))
-            assert 0 == len(list(self.project.find_job_ids({"a": 5})))
-            assert 1 == len(list(self.project.find_job_ids({"sp.a": 0})))
-            assert 0 == len(list(self.project.find_job_ids({"sp.a": 5})))
-            assert 1 == len(list(self.project.find_job_ids(doc_filter={"b": 0})))
-            assert 0 == len(list(self.project.find_job_ids(doc_filter={"b": 5})))
-            assert 1 == len(list(self.project.find_job_ids({"doc.b": 0})))
-            assert 0 == len(list(self.project.find_job_ids({"doc.b": 5})))
-            assert 1 == len(list(self.project.find_job_ids({"a": 0, "doc.b": 0})))
-            assert 1 == len(list(self.project.find_job_ids({"sp.a": 0, "doc.b": 0})))
-            assert 0 == len(list(self.project.find_job_ids({"sp.a": 0, "doc.b": 5})))
-            assert 0 == len(list(self.project.find_job_ids({"sp.a": 5, "doc.b": 0})))
-            assert 0 == len(list(self.project.find_job_ids({"sp.a": 5, "doc.b": 5})))
```

Comment:
There are some test cases for unified queries like `{"sp.a": 5, "doc.b": 5}` shown in this function that are not in `test_find_jobs` below. I moved those test cases to `test_find_jobs` so that we retain coverage of those code paths.